libs/sdk-ui-dashboard/src/model/state/config/configSelectors.ts

```diff
@@ -185,3 +185,39 @@ export const selectPlatformEdition = createSelector(selectConfig, (state) => {
 export const selectEnableCompanyLogoInEmbeddedUI = createSelector(selectConfig, (state) => {
     return state.settings?.enableCompanyLogoInEmbeddedUI ?? false;
 });
+
+/**
+ * Returns whether the export to pdf is enabled.
+ *
+ * @alpha
+ */
+export const selectEnableKPIDashboardExportPDF = createSelector(selectConfig, (state) => {
+    return state.settings?.enableKPIDashboardExportPDF ?? false;
+});
+
+/**
+ * Returns whether the insecure javascript events are enabled.
+ *
+ * @alpha
+ */
+export const selectInsecureJavascriptEventsEnabled = createSelector(selectConfig, (state) => {
+    return state.settings?.insecureJavascriptEventsEnabled ?? false;
+});
```

Comment:
i suggest to remove this one; the entire settings can be selected and the flag tested in app

Reply:
Removed